xmtp_mls/src/groups/mls_sync.rs

```diff
@@ -1323,6 +1351,71 @@ where
         Ok(Some(msg))
     }
 
+    async fn process_group_message_error_for_fork_detection(
+        &self,
+        provider: &XmtpOpenMlsProvider,
+        message: &GroupMessageV1,
+        error: &GroupMessageProcessingError,
+    ) -> Result<(), GroupMessageProcessingError> {
+        let group_id = message.group_id.clone();
+
+        if let OpenMlsProcessMessage(ProcessMessageError::ValidationError(
+            ValidationError::WrongEpoch,
```

Comment:
we're catching wrong epoch here, but not decryption error for the right epoch. Were we able to figure out a place to catch those areas as well, so that we can log it? Is it possible to do in this function too

Reply:
OK, lets create an issue to cover the AEAD fork errors so we don't lose track of it as well